docs/en/latest/deployments/kubesphere.md

```diff
@@ -50,6 +50,19 @@ helm install apisix apisix/apisix \
   --namespace ingress-apisix \
   --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
 kubectl get service --namespace ingress-apisix
+
+# Due to network reasons, you may have problems when deploying. You can consider using an offline method. Here is an example:
```

Comment:
Could you split this to a separate code block? It would make it easy for the users to copy either of the code blocks. Since it is in the same code block now, both will be copied.

You can use a `:::tip` admonition as shown in this document below to highlight this point.

The sentence could be:

"If you have problems deploying APISIX due to network reasons, you can download, copy, and install APISIX as shown below:"

Reply:
good.